slides/_formatting.qmd

```diff
@@ -135,13 +147,33 @@ def long_func(
 ::::
 
 
-## PEP8 & Formatting - Black {.smaller}
+<!-- ------------------------------------------------------------------------------ -->
+
+
+## PEP8 & Formatting - Ruff {.smaller}
+
+- Also runs on jupyter notebooks.
+
+- Highly configurable via configuration files.
 
 - I suggest incorporating into your projects now
-  - Well suited to incorporation into continuous integration of git hooks.
-  - Widely-used standard^[[Black - used by](https://github.com/psf/black#used-by)]
+  - Widely-used standard^[[Who's Using Ruff?](https://github.com/astral-sh/ruff#whos-using-ruff)]
+  - Plugins/lsp available for many editors.
+  - Well suited to incorporation into continuous integration through workflows and git hooks.
 
-- A version for jupyter notebooks exists.
+
+<!-- ------------------------------------------------------------------------------ -->
+
+
+## Other languages {.smaller}
```

Comment:
Should we maybe also mention alternatives for Python, such as Black? (And then we might or might not want to keep the references in the bib file)

Reply:
Done, although without reference as the other examples don't have associated references.